ping_federate/assets/dashboards/ping_federate_admin.json

```diff
@@ -47,8 +47,8 @@
         "margin": "md",
         "sizing": "contain",
         "type": "image",
-        "url": "https://support.pingidentity.com/s/sfsites/c/sfc/servlet.shepherd/version/download/0688Z00000QVi33QAD",
-        "url_dark_theme": "",
+        "url": "https://allvectorlogo.com/img/2016/06/pingfederate-logo.png",
```

Comment:
do we want to update images to here?

Reply:
Yes, We have updated the image to support it in dark theme.